src/main/java/mate/academy/dao/UserDao.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserDao {
+    User add(User user);
+
+    Optional<User> get(String email);
```

Comment:
it's better to use more informative names for methods (ex `getByEmail` or `findByEmail`)